index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
You've linked the CSS file in your HTML file, but CSS is still empty. Kindly finish styling before making a PR, it would make it possible to evaluate your code's outcome.